### PR TITLE
Current status

### PR DESCRIPTION
**Pull request by aTonyXiao**
add API for changing current status

### CODE REVIEW

File: bugswarm/common/rest_api/database_api.py [hunk 1]
```diff
@@ -263,6 +263,22 @@ def set_artifact_stability(self, image_tag: str, stability) -> Response:
         updates = {'stability': stability}
         return self._patch(DatabaseAPI._artifact_image_tag_endpoint(image_tag), updates)
 
+    def set_artifact_current_status(self, image_tag: str, status: str, date: str) -> Response:
+        """
+        Update the current status of an artifact.
+
+        :param image_tag: The image tag identifying the artifact to update.
+        :param status: Unreproducible/Reproducible/Broken/Flaky.
+        :param date: the date of artifacts status changed
```
Comment (dtomassi) [inline bugswarm/common/rest_api/database_api.py]: What is the format of the date? I think that would be good addition to the docstring.

File: bugswarm/common/rest_api/database_api.py [hunk 2]
```diff
@@ -263,6 +263,22 @@ def set_artifact_stability(self, image_tag: str, stability) -> Response:
         updates = {'stability': stability}
         return self._patch(DatabaseAPI._artifact_image_tag_endpoint(image_tag), updates)
 
+    def set_artifact_current_status(self, image_tag: str, status: str, date: str) -> Response:
```
Comment (dtomassi) [inline bugswarm/common/rest_api/database_api.py]: Should we validate the status (make sure it is of the four different options) and the date?.

Comment (aTonyXiao) [reply bugswarm/common/rest_api/database_api.py]: We have the validations in database schema. see [here](https://github.com/BugSwarm/database/blob/0f14dde8d091c89b82900b4479120a95037695ec/database/schema/artifact_schema.py#L87).

File: bugswarm/common/rest_api/database_api.py (hunk 1 above)
Comment (aTonyXiao) [reply bugswarm/common/rest_api/database_api.py]: Good idea

**Pull request merged**